dtfs-central-api/api-tests/v1/utilisation-reports/put-utilisation-report-status.api-test.ts

```diff
@@ -5,7 +5,7 @@ import { SqlDbHelper } from '../../sql-db-helper';
 
 console.error = jest.fn();
 
-describe('/v1/utilisation-reports/set-status', () => {
+describe('PUT /v1/utilisation-reports/set-status', () => {
   const setStatusUrl = '/v1/utilisation-reports/set-status';
```

Comment:
Nitpick, could we pull this out into a BASE_URL var and use it in the description text too (as with the other test files)

Reply:
Done :)

Reply:
Sorry as in once we've pulled BASE_URL out could we use that both in the description text (done now) and also replace all `setStatusUrl` references with BASE_URL and remove the `setStatusUrl` var